core/src/main/resources/error/error-classes.json

```diff
@@ -581,6 +581,11 @@
     ],
     "sqlState" : "42818"
   },
+  "INCOMPATIBLE_COLUMN_TYPE" : {
+    "message" : [
+      "<operator> can only be performed on tables with compatible column types. The <ci> column of the <ti> table is <dt1> type which is not compatible with <dt2> at the same column of the first table.<hint>."
```

Comment:
shall we rename parameters? `ci` and `ti` looks confusing.

Reply:
Sounds good! Let me update it

Reply:
Updated!